### DISCUSSION TITLE
Cumulative sector times / current lap time

### DISCUSSION THREAD

**Discussion by xiphiasphotography** [f1multiviewer/issue-tracker#508] [Feature requests]
It would be nice of the total sector times could be displayed in a column. A live timer would be even better, especially in on-board view like F1 on-boards.
S1, S1 + S2, S1 + S2 + S3 as one cumulative time in the table which resets to S1 time when next S1 has finished. Same as F1.

In the on-board a live lap time would be nice, especially for qualifing, but could be hard to get right.

**Comment by jeppeolesen** [comment]
Redirecting this to #349
